src/lib/SiteTypes/src/sitetypes/tj.jl

```diff
@@ -183,7 +183,13 @@ function op!(Op::ITensor, ::OpName"Sx", ::SiteType"tJ", s::Index)
   Op[s' => 2, s => 3] = 0.5
   return Op[s' => 3, s => 2] = 0.5
 end
-
+function op!(Op::ITensor, ::OpName"Sy", ::SiteType"tJ", s::Index)
+    Op[s' => 2, s => 3] = -1im*0.5
+    return Op[s' => 3, s => 2] = 1im*0.5
+  end
```

Comment:
```suggestion
  Op[s' => 2, s => 3] = -0.5im
  return Op[s' => 3, s => 2] = 0.5im
end
```
Just a bit shorter, also fixes the formatting.